src/core/worker.c

```diff
@@ -205,6 +205,8 @@ QuicWorkerQueueConnection(
 
     QuicDispatchLockRelease(&Worker->Lock);
 
+    QuicPerfCounterIncrement(QUIC_PERF_COUNTER_CONN_QUEUE_DEPTH);
```

Comment:
This isn't the right place. Notice above where the connection is actually added to the list and the ref count is added. This perf counter should pretty much match up with ref count add/release calls.

Reply:
Since, in the other place, we decided to not have the counter increment inside the lock, I'm going to set a boolean when the connection is queued, and wrap this counter in an if-statement.